containers/streamsets/credential-stores.properties

```diff
@@ -49,7 +49,7 @@ credentialStore.jks.config.keystore.type=PKCS12
 
 # The location of the Java keystore. Specify an absolute path or a path relative to the
 # $SDC_CONF directory.
-credentialStore.jks.config.keystore.file=jks-credentialStore.pkcs12
+credentialStore.jks.config.keystore.file=/data/security/jks-credentialStore.pkcs12
```

Comment:
there won't be any errors if /data/security directory doesn't exist?

Reply:
Will be an error. That is what I've added one more step with mkdir command to a Dockerfile

Reply:
I've also tried to specify env variable instead of `/data/security/`, but it did not work. You can see attached screenshot in a Jira ticket